students/Gregg/lesson3/mailroom.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python3
+"""Use current knowledge to set up an interactive program"""
+
+donations = {
+    "William Gates, III": [3, 5, 7],
+    "Mark Zuckerberg": [4.50, 8, 2],
+    "Jeff Bezos": [7.77],
+    "Paul Allen": [3.6, 4.5],
+    "bob": [.01]
+}
```

Comment:
In lesson3 you should use data structure learned so far: tuples and lists - this allows you to practice those data structures 🙂